Sources/UI/ActionButton/ActionButton.swift

```diff
@@ -33,25 +39,56 @@ public struct ActionButton: View {
   }
   
   public var body: some View {
-    GeometryReader { geo in
-      ZStack(alignment: getAlignment(for: properties.extraImage)) {
-        ButtonView(size: geo.size, action: buttonAction, properties: properties)
-        ExtraImageView(imageAlignment: properties.extraImage, tintColor: properties.textColor, size: geo.size)
+    Button(action: {
+      properties.actionHandler?()
+    }) {
+      HStack {
+        properties.leftImage?.image
+          .resizable()
+          .aspectRatio(contentMode: .fit)
+          .frame(width: properties.leftImage?.size.width,
+                 height: properties.leftImage?.size.height)
+        Spacer()
+        HStack {
+          properties.titleLeftImage?.image
```

Comment:
Do we need both "leftImage" and "titleLeftImage"? 🤔

Reply:
`leftImage` is the one on the left almost bordering the edge and the `titleLeftImage` is the one right next to the title. 
They are in different stacks. We will have cases where both will be used maybe not simultaneously, but I didn't want to constraint to using just one at a time and left all options open.

For eg. we could have a button with a left image and then an arrow next to the button title. In this case we need both of them.
Even though the file doesn't have a button like that I wanted to leave this kind of flexibility.

Reply:
@ArlindDushi should we rename `titleLeftImage` to `titleImage`?